docs/source/development/commands_reference.md

```diff
@@ -109,14 +109,14 @@ Returns output similar to the following, depending on the version of Kedro used
 | |/ / _ \/ _` | '__/ _ \
 |   <  __/ (_| | | | (_) |
 |_|\_\___|\__,_|_|  \___/
-v0.19.7
+v0.19.8
 
 Kedro is a Python framework for
 creating reproducible, maintainable
 and modular data science code.
 
 Installed plugins:
-kedro_viz: 4.4.0 (hooks:global,line_magic)
+kedro_viz: 9.2.0 (hooks:global,line_magic)
```

Comment:
Oh wow have we just neglected updating this?! 👀

Reply:
@merelcht Yeah I don't think we have the habit to make sure this is up to date, but this is surely quite outdated🤓